pages/plugins.jsx

```diff
@@ -33,23 +21,31 @@ const integrations = [
   },
 ]
 
-const Plugins = () => {
-  return (
-    <>
-      <h1>Plugins</h1>
-      <p>Integrate our powerful services into your website.</p>
-      <div className={pluginsClassNames.plugins}>
-        {integrations.map(i => (
-          <Plugin
-            key={i.name}
-            name={i.name}
-            abstract={i.abstract}
-            description={i.description}
-          />
-        ))}
-      </div>
-    </>
-  )
+class Plugins extends React.Component {
+  static async getInitialProps({ initialStoreData }) {
+    const { statusCode, ...plugins } = await API.getIntegrations(140)
+    initialStoreData.plugins = plugins
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
You can disable this globally if you experience this problem again.

Maybe an overriding rule for `pages/*.jsx`

Reply:
I was considering it but I decided to give it another whirl.